src/main/resources/lang/locale-zh.properties

```diff
@@ -0,0 +1,9 @@
+language=Chinese
+disambiguation=disambig,hndis,Geodis,otheruses,Otheruses,distinguish,Distinguish,otheruseslist,Otheruseslist,About,about,SelfDistinguish
```

Comment:
I would expect the member on the right to be written in Chinese :) (have a look at the other locales ;))